sys/arvo.hoon

```diff
@@ -502,101 +515,232 @@
     $(ova (weld p.nyx ova), mor (weld q.nyx t.mor))
   --
 --
-=+  pit=`vase`!>(.)                                     ::
-=+  bud=pit                                             ::  becomes tang
-::  =+  vil=(viol p.bud)                                ::  cached reflexives
-=+  vil=(viol p.bud)                                    ::  cached reflexives
-=|  $:  lac/?                                           ::  laconic bit
-        eny/@                                           ::  entropy
+=<  ::  Arvo larval stage
+    ::
+    ::    The true Arvo kernel knows who it is. It should not *maybe*
+    ::    have an identity, nor should it contain multitudes. This outer
+    ::    kernel exists to accumulate identity, entropy, and the
+    ::    standard library. Upon having done so, it upgrades itself into
+    ::    the true Arvo kernel. Subsequent upgrades will fall through
+    ::    the larval stage directly into the actual kernel.
+    ::
+    ::    For convenience, this larval stage also supports hoon compilation
+    ::    with +wish and vane installation with the %veer event.
+    ::
+    =/  pit=vase  !>(..is)
+    =|  $:  ::  who: our identity once we know it
+            ::  eny: entropy once we learn it
+            ::  bod: %zuse once we receive it
+            ::
+            who=(unit ship)
+            eny=(unit @)
+            bod=(unit vase)
+        ==
+    ::  larval Arvo structural interface
+    ::
+    |%
+    ++  come  ^come                                     ::  22
+    ++  load  ^load                                     ::  46
+    ::
+    ::  XX my preference would be for +peek to no-op here
```

Comment:
Why would we want +peek to no-op? Seems useful

Reply:
The larval stage doesn't let any events through to the vanes. Are there useful scrys into vanes with default state?

Reply:
Then, I say go for it. I want us to err more on the side of removing unused code in general.